.github/no-response.yml

```diff
@@ -0,0 +1,13 @@
+# Configuration for probot-no-response - https://github.com/probot/no-response
```

Comment:
i think I've seen bots that post a comment first with a warning that the issue will be closed if it stays without a response for next X days, IMHO would be better to use such behaviour, because sometimes author has provided enough info, but we didnt have time to look into it - so posting a warning before closing gives author opportunity to keep the thread alive

Reply:
[probot/stale](https://github.com/probot/stale)? I was intending to use this by commenting on issues and asking for more information and then adding the label, this one doesn't do anything unless you add the label. I think we could add that one as well but I think this one is still useful because it's better for closing things that don't have necessary information faster.

Reply:
not sure about all intricacies of those 2, after making a rly quick look - I think probot/stale is better (more user friendly)

not sure how they both can be used at the same time (stale seems to overlap functionalities with no-response), but as mentioned - I have not thought this through really :p if you believe both are useful or one is of them is better then just go with ur gut